test/iterator.js

```diff
@@ -289,7 +289,7 @@ metatests.test('Iterator.some with thisArg', test => {
     },
   };
 
-  test.assertNot(iter(array).every(obj.predicate, obj));
+  test.assertNot(iter(array).some(obj.predicate, obj));
```

Comment:
What do you think about setting the `max` value to `2` so that test would fail with `every` and pass only with `some`?

Reply:
Ok.